optaplanner-test/pom.xml

```diff
@@ -63,6 +63,7 @@
     <dependency>
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
```

Comment:
Quickstarts failed because of this and I wonder...
Is this a backwards incompatible change? Can we afford this?
I'd vote to merge it - people should be responsible for their own dependencies.

Reply:
If we don't need it for runtime anymore, I agree. Whoever depends on `assertj` should not rely on us to provide it transitively.